config/config.go

```diff
@@ -660,6 +663,29 @@ func (c *Config) ToRouteSrvOptions() routesrv.Options {
 		// Auth:
 		EnableOAuth2GrantFlow: c.EnableOAuth2GrantFlow,
 		OAuth2CallbackPath:    c.Oauth2CallbackPath,
+
+		// swarm:
```

Comment:
we don't need the swarm options (timeouts and access to redis are only interesting in the data plane)

Reply:
Can you check now? Removed redisClient connection.